src/Analysis/GWASWizard/GWASCaseControl.jsx

```diff
@@ -190,9 +195,21 @@ const GWASCaseControl = ({ resetGWASType, refreshWorkflows }) => {
           <React.Fragment>
             {selectedCovariates.length === 0
                 && (
-                  <div className='GWASUI-emptyTable'>
-                    <span>No covariates to review.</span>
-                  </div>
+                  <React.Fragment>
+                    <div className='GWASUI-mainTable GWASUI-spinnerContainer'>
+                      <Table
+                        dataSource={[{ placeholder: 'No covariate missingness to review' }]}
```

Comment:
The original text was more accurate, I propose we keep that.

Reply:
ok